charts/datadog/templates/_helpers.tpl

```diff
@@ -767,3 +767,22 @@ true
 false
 {{- end -}}
 {{- end -}}
+
+
+{{/*
+Create RABCs for custome resources
+*/}}
+{{- define "orchestratorExplorer-config-crs" -}}
+
+{{- range $cr := .Values.datadog.orchestratorExplorer.customResourcesCollection }}
+- apiGroups:
```

Comment:
just confirming that versions aren't necessary as part of the RBACs?

Reply:
they are not necessary